workshops/personal_website/README.md

```diff
@@ -0,0 +1,239 @@
+# Personal Website
+
+In this workshop, you will learn how to make a website from scratch.
+
+Here is the final product you will end up with:
+
+![](http://placehold.it/200?text=Preview)
+
+> Open the [live demo](#TODO).
+>
+> View [the code](#TODO).
+
+## Part I: Setup Your Tools
+
+### GitHub
+
+- What is GitHub?
+- What are we using GitHub for?
+- Let's setup your GitHub account.
+- Write down your email, username, and password somewhere you won't forget
+  (you'll need it later in this tutorial and later in the semester).
+
+### Cloud9
+
+- What is Cloud9?
+- What are we using Cloud9 for?
+- Let's create a Cloud9 Account.
+- Let's create and setup a Cloud9 workspace.
+- Write down your email, and password somewhere you won't forget.
+  (you'll be using it for the rest of the semester).
+
+## Part II: HTML
+
+- What is HTML?
+- What are we using HTML for?
+
+### Step 1: Create a folder for your personal website
+
+- Let's create a website
+
+### Step 2: Creating your HTML file
+
+### Step 3: Add the standard HTML template
+
+- Let's paste the below code
```

Comment:
Should we have them type out this instead? Might be a good idea to start the habit of typing from the very first example.